pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -275,11 +303,26 @@ func createVolumeMount(ledgerDirs []string, journalDirs []string, indexDirs []st
 		}
 		volumeMounts = append(volumeMounts, v)
 	}
-	v := corev1.VolumeMount{
-		Name:      heapDumpName,
-		MountPath: heapDumpDir,
```

Comment:
why are we removing `heapDumpDir`. If there is no emptydir volumes are mentioned in options, it will not use the deafult  `heapDumpDir`

Reply:
I added 2 options: `hostPathVolumeMounts` and `emptyDirVolumeMounts`